views/apartments/tenants.erb

```diff
@@ -0,0 +1,28 @@
+<script type="text/javascript">
+  $(document).ready(function(){
+      $("button").click(function(){
+        if (this.innerHTML == "Add A Tenant"){
+          $("form").hide(1000);
+          $("#create").toggle('slow','swing');
+        }
+      });
+  });
```

Comment:
This is a nice little snippet. It would better to create a js file for it in `public/js/coolSnippet.js` and link to it. That way you don't have to duplicate it on multiple views and you can better separate your concerns.